ci/bamboo/build-manylinux-prototype.sh

```diff
@@ -1,101 +0,0 @@
-#!/bin/bash
```

Comment:
After a little more consideration, my thoughts are below. @vitaly-krugl can we discuss when you get into the office tomorrow morning? @oxtopus - I'll run things by you afterwards.

- I like this separation of wheel building and wrappers for building the wheel locally vs. in CI.
- My biggest concern is the same as Austin's, which is keeping this stuff simple. I had separate scripts for release and debug previously because argument parsing seemed unnecessarily complex and  I don't like the indirect relationship of environment variables.
- That said, I'm fine with trying to get this all implemented cleanly. With previous CI scripts we started enforcing that they were implemented in Python since bash can be difficult to read and maintain. I'm not sure if this makes sense here or not.
- The build_manylinux_wheel.sh script isn't really specific to manylinux. It uses that name in a couple places but it is mostly just a bash script for building the wheel and when you run it inside the manylinux Docker container then you get a manylinux wheel.